### PR TITLE
React-router documentation for github pages

### PR DESCRIPTION
**Pull request by arbob**
I encountered a pretty frustrating problem while deploying it to gh-pages. most of it was due to github not detecting the routes i've added. I tried searching on the internet regarding the solution and after 1-2 hours of trial and error came across this
https://github.com/facebook/create-react-app/issues/1765
solution by react team to somehow use tricks to bypass this problem.
I wonder if people here got into the same problem while deploying router based react page. I suggest Odin team to add some steps or instructions to better deploy router based projects on the assignment page such that noone else has to suffer the same. 
However, @todoroki in discord room suggested to create a PR for the same.

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).

#### 1.Describe the changes made:

...your text here

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

File: javascript/react-js/project-shopping-chart.md [hunk 1]
```diff
@@ -14,5 +14,5 @@ This application should also have at least two routes to give you a chance to ma
 6. Build individual card items for each of your products. Display an input field on it, which lets a user manually type in how many items they want to buy. Also, add an increment and decrement button next to it for fine-tuning. You can also display a title for each product as well as an "Add To Cart" button.
 7. Once a user has submitted their order, the amount on the cart itself should adjust.
 8. As usual, style your application so you can show it off!
-9. Lastly, push the project to GitHub!
+9. Lastly, push the project to GitHub! <a href="https://create-react-app.dev/docs/deployment/#github-pages">Follow this link for instructions on deploying router based react pages on GitHub.</a>.
```
Comment (twalton83) [inline javascript/react-js/project-shopping-chart.md:17]: Can we instead say “Follow this link for instructions on how to deploy React applications with client-side routing to Github Pages.”

**Review by twalton83 — CHANGES_REQUESTED** [review]

File: javascript/react-js/project-shopping-chart.md (hunk 1 above)
Comment (arbob) [reply javascript/react-js/project-shopping-chart.md:17]: That works.

**Review by arbob — COMMENTED** [review]
Changing comment to "Follow this link for instructions on how to deploy React applications with client-side routing to Github Pages."

File: javascript/react-js/project-shopping-chart.md [hunk 2]
```diff
@@ -14,5 +14,5 @@ This application should also have at least two routes to give you a chance to ma
 6. Build individual card items for each of your products. Display an input field on it, which lets a user manually type in how many items they want to buy. Also, add an increment and decrement button next to it for fine-tuning. You can also display a title for each product as well as an "Add To Cart" button.
 7. Once a user has submitted their order, the amount on the cart itself should adjust.
 8. As usual, style your application so you can show it off!
-9. Lastly, push the project to GitHub!
+9. Lastly, push the project to GitHub! <a href="https://create-react-app.dev/docs/deployment/#github-pages">Follow this link for instructions on how to deploy React applications with client-side routing to Github Pages.</a>.
```
Comment (twalton83) [inline javascript/react-js/project-shopping-chart.md:17]: We use markdown, so it should look like this \[text here\]\(link here\).

**Review by twalton83 — CHANGES_REQUESTED** [review]

File: javascript/react-js/project-shopping-chart.md (hunk 2 above)
Comment (arbob) [reply javascript/react-js/project-shopping-chart.md]: changed.

**Comment by arbob** [comment]
Any updates on this? @twalton83

**Review by twalton83 — APPROVED** [review]

**Pull request merged**